ci-operator/step-registry/idp/external-oidc/keycloak/aks/server/idp-external-oidc-keycloak-aks-server-ref.yaml

```diff
@@ -0,0 +1,22 @@
+ref:
+  as: idp-external-oidc-keycloak-aks-server
+  from_image:
+    namespace: ocp
+    name: "4.19"
+    tag: upi-installer
+  commands: idp-external-oidc-keycloak-aks-server-commands.sh
+  env:
+  - name: HYPERSHIFT_BASE_DOMAIN
+    default: "ci.azure.devcluster.openshift.com"
```

Comment:
ci-operator/step-registry/hypershift/azure/aks/oidc/hypershift-azure-aks-oidc-workflow.yaml also defines `HYPERSHIFT_BASE_DOMAIN` but the default is different. Can they be consistent. It may reads confusing at first glance for others?

Reply:
I just removed the HYPERSHIFT_BASE_DOMAIN in ypershift-azure-aks-external-oidc workflow, let it use the default value

Reply:
OK